popup_filter_build.js

```diff
@@ -0,0 +1,54 @@
+const { promises: fs } = require('fs');
+
+/**
+ * Path to the filter file
+ * @type {string}
+ */
+const filterPath = process.argv[2];
+
+/**
+ * Function to convert modifiers in a rule
+ * @param {string} rule - The rule to modify
+ * @param {string} modifier - The new modifier
+ * @param {string} [modifierValue] - The value for the modifier
+ * @returns {string} - Modified rule
+ */
+const convertModifier = (rule, modifier, modifierValue) => {
```

Comment:
If you used `RemoveModifiers`, you could simply append `$dnsrewrite` instead of messing with regular expressions.